Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official Node.js 18 base image
+FROM node:18
+
+# Set the working directory inside the container
+WORKDIR /usr/src/app
+
+# Copy the src folder
+COPY src ./src
```

Comment:
It's must be replaced with copying the `build` command folder, and note the `build` folder is the artifact of compiling/building the typescript project

Reply:
@aminlatifi Sorry I don't think I'm able to follow, not much experience in building from source, can you share the Dockerfile you think is working and we can close the PR?

Reply:
I've modified the CI and the Dockerfile in the latest commit to accommodate the latest suggested changes. Please check